.gitattributes

```diff
@@ -22,7 +22,7 @@
 _config.yml export-ignore
 codecov.yml export-ignore
 dev/kyuubi-tpcds/ export-ignore
-docs/ export-ignore
+docs/imgs export-ignore
```

Comment:
How can we use the imgs files?

Reply:
It's part of the doc site, and should we suppose that users deploy the doc site from source release tarball?

Reply:
Have a one-by-one check on all images, removed all unused images, and now I think there is no license issue on `docs/imgs`, thus removed this exclude rule.